src/MobileUI/ViewModels/EarnDetailsViewModel.cs

```diff
@@ -157,7 +149,7 @@ private async Task SubmitResponses()
 
             if (allQuestionsAnswered)
             {
-                var popup = new QuizResultPending();
+                var popup = new QuizResultPendingPage(new QuizResultPendingViewModel());
```

Comment:
I wonder if we can use DI for popups...? 🤔

Reply:
I tried, but could not quickly find a solution.